lib/pgBackRest/Backup/Backup.pm

```diff
@@ -379,7 +387,7 @@ sub processManifest
             $iHostConfigIdx, $strQueueKey, $strRepoFile, OP_BACKUP_FILE,
             [$strDbFile, $strRepoFile, $lSize,
                 $oBackupManifest->get(MANIFEST_SECTION_TARGET_FILE, $strRepoFile, MANIFEST_SUBKEY_CHECKSUM, false),
-                optionGet(OPTION_CHECKSUM_PAGE) ? isChecksumPage($strRepoFile) : false, $bCompress,
+                optionGet(OPTION_CHECKSUM_PAGE) ? isChecksumPage($strRepoFile) : false, $strBackupLabel, $bCompress,
```

Comment:
Shouldn't.  Before this function was always copying to tmp/stanza but now it needs to know the backup label.